doc/config.rst

```diff
@@ -0,0 +1,670 @@
+
+type: **RubinOptics**
+^^^^^^^^^^^^^^^^^^^^^
+Photons ray-traced though the Rubin optical system.
+
+Required keywords to set:
+""""""""""""""""""""""""""
+
+    * ``boresight`` = *RaDec_value* (default = None) The CelestialCoord of the boresight of the observation.
+    * ``camera`` = *str_value* (default = None) The name of the camera to use.
+
+type: **RubinDiffractionOptics**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Ray-traced photons including the effects of diffraction when passing through edges like the telescope spiders.
+
+Required keywords to set:
+"""""""""""""""""""""""""
+
+    * ``boresight`` = *RaDec_value* (default = None) The CelestialCoord of the boresight of the observation.
+    * ``camera`` = *str_value* (default = None) The name of the camera to use.
+    * ``azimuth`` = *float_value* (default = None) The azimuth angle in degrees.
+    * ``altitude`` = *float_value* (default = None) The altitude angle in degrees.
+
+
+Optional keywords to set:
+""""""""""""""""""""""""""
+
+    * ``latitude`` = *bool_value* (default = True) The latitude of the observatory.
+    * ``disable_field_rotation`` = *bool_value* (default = False) Do not consider the effect of the rotation of the camera relative to the spiders of the telescope during the exposure.
+
+type: **RubinDiffraction**
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Diffractive effects only during the FFT.
+
+Required keywords to set:
+"""""""""""""""""""""""""
+
+    * ``azimuth`` = *float_value* (default = None) The azimuth angle in degrees.
+    * ``altitude`` = *float_value* (default = None) The altitude angle in degrees.
+    * ``latitude`` = *bool_value* (default = True) The latitude of the observatory.
+
+Optional keywords to set:
+""""""""""""""""""""""""""
+
+    * ``disable_field_rotation`` = *bool_value* (default = False) Do not consider the effect of the rotation of the camera relative to the spiders of the telescope during the exposure.
+
+Output types
+============
+
+The output field is used to specify where to write output files and what format they should be.  There are several possibilities, including FITS files before and after electronics readout, and various types of truth information.  *imSim* adds the ``LSST_CCD`` type. It  understands how to write "eimage" files which are true representations of the electrons in the CCD including signals from the objects and cosmic rays with important physics effects such as the brighter-fatter effect and tree-rings applied.
+
+It can also write "amp" files. These are fully readout electronics files with one amplifier per FITS HDU with all of the proper headers needed to be processed by the Rubin Science Pipelines.  Both of these output formats can be examined with standard tools such as *ds9*.
+
+There are also several extra outputs available to the user including a centroid file containing the true position of the rendered sources, a list of optical path differences in the optical system, and a map of surface figure errors.
```

Comment:
The centroid file isn't an imsim-special thing.  (And you don't document it below.)  Maybe don't mention it here, so users don't go looking for it.  Or mention it, but say we just use the standard GalSim "truth" extra output type.

Reply:
Clarified.